src/main/java/com/github/vkorobkov/jfixturescmd/cmd/CmdParser.java

```diff
@@ -22,30 +29,46 @@ public void parse(String[] args) {
             if (cmdArgs.isHelp()) {
                 jCommander.usage();
             } else {
-                generateSql();
+                generateData();
             }
         } catch (Exception e) {
             ExceptionHandler.handleException(e);
             jCommander.usage();
         }
     }
 
+    @SneakyThrows
+    private void generateData() {
+        val destination = cmdArgs.getDestination();
+        val fixturesFolder = cmdArgs.getSource();
+        val type = cmdArgs.getDestinationType();
+
+        log.info("Fixtures folder: " + fixturesFolder);
+        if (Objects.equals(type.toLowerCase(), "xml")) {
+            log.info("Export type: XML");
+            JFixtures.xml(fixturesFolder).toFile(destination);
+        } else {
+            log.info("SQL type: " + type);
+            JFixtures.byDialect(fixturesFolder, convertToSqlType(type)).toFile(destination);
+        }
+        log.info("\nSUCCESS (destination file: " + destination + ")\n");
+    }
+
+    @SneakyThrows
+    private SqlType convertToSqlType(final String value) {
+        Optional<SqlType> sqlType = EnumUtil.valueOf(SqlType.class, value.toUpperCase());
+        return sqlType.orElseThrow(() -> {
+                    String availableTypes = Arrays.toString(SqlType.values()).toLowerCase();
+                    throw new ParameterException("SQL type '" + value + "' is not valid. "
```

Comment:
This exception should say that "possible output types are: [mysql, mssql, sql99, xml]" (the phrase is bad but the meaning is right). Currently the XML type does not present which is wrong.